components/brave_ads/rust/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "brave-ads-cxx"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+cxx = "1.0"
+lazy_static = "1.4.0"
+regex = "1.5.5"
```

Comment:
`brave_news` uses the regex version `1.5.5`. So I decided to use the same version to avoid additional dependencies.
I think it is better to bump the version as a separate PR for all regex deps.

Reply:
Fine with me. Avoiding additional dependencies is what we want.

To be clear, brave news *specifies* `1.5.5` but actually uses version `1.7.0`. This is controlled by `brave/build/rust/Cargo.lock`. Higher minor numbers are considered backward-compatible under [semver](https://semver.org/), so this is considered to match the specification.